tests/baselines/reference/moduleNoneDynamicImport(target=es2015).js

```diff
@@ -0,0 +1,11 @@
+//// [tests/cases/compiler/moduleNoneDynamicImport.ts] ////
+
+//// [a.ts]
+const foo = import("./b");
+
+//// [b.js]
+export default 1;
+
+
+//// [a.js]
+const foo = Promise.resolve().then(() => require("b"));

```

Comment:
Isn't the request to make sure this preserves the `import`?

Reply:
@DanielRosenwasser This file is related to target `es2015`. I thought the import should be preserved for `target >= es2020` and `module None`. Or am I wrong?